components/org.wso2.carbon.identity.oauth.dcr/src/main/java/org/wso2/carbon/identity/oauth/dcr/service/DCRMService.java

```diff
@@ -809,4 +831,23 @@ private ServiceProvider cloneServiceProvider(ServiceProvider serviceProvider) {
         ServiceProvider clonedServiceProvider = gson.fromJson(gson.toJson(serviceProvider), ServiceProvider.class);
         return clonedServiceProvider;
     }
+
+    /**
+     * Validate SSA signature using jwks_uri.
+     * @param softwareStatement Software Statement
+     * @throws DCRMClientException
+     * @throws IdentityOAuth2Exception
+     */
+    private boolean validateSSASignature(String softwareStatement) throws DCRMClientException, IdentityOAuth2Exception {
+
+        SignedJWT signedJWT = null;
+        try {
+            signedJWT = SignedJWT.parse(softwareStatement);
+        } catch (ParseException e) {
+            throw new DCRMClientException(DCRMConstants.ErrorCodes.INVALID_SOFTWARE_STATEMENT,
```

Comment:
e is missing.

Reply:
fixed in 6ee33bbad29564b167d3a5cd953cb081c4755c0f